lago/sysprep.py

```diff
@@ -117,6 +117,14 @@ def config_net_interface_dhcp(iface, hwaddr):
     )
 
 
+def edit(filename, expression):
+    editstr = '%s:""%s""' % (filename, expression)
```

Comment:
Just to make sure, here there are two pairs of double quotes, is that what you intended?